TYPO3.Neos/Classes/TYPO3/Neos/Controller/Backend/ContentController.php

```diff
@@ -258,7 +259,7 @@ protected function getImagePreviewData(Image $image)
                 'height' => $thumbnail->getHeight()
             ];
         }
-        $thumbnail = $this->thumbnailService->getThumbnail($image, $this->thumbnailService->getThumbnailConfigurationForPreset('TYPO3.Neos:InspectorImage'));
+        $thumbnail = $this->thumbnailService->getThumbnail($image, new ThumbnailConfiguration(null, 300, null, 300));
```

Comment:
Not an easy choice.
I went for the ``TYPO3.Neos:Thumbnail`` it's a little bit too small for the container, but still better than loading an image ten times too big IMHO.